packages/design-system/scripts/cmsds-migrate/configs/html-button-migration.mjs

```diff
@@ -0,0 +1,35 @@
+export default {
+  description:
+    'Updates button classes for latest button architecture as of 08/03/2022.\n  - Replaces ds-c-button--primary/success with ds$-c-button--solid\n  - Replaces ds-c-button--transparent with ds-c-button--link\n  - Replaces ds-c-button--inverse with ds-c-button--on-dark\n  - Flags ds-c-button and ds-c-button--hover/active/focus as deprecated',
```

Comment:
Looks like there might be a typo here

```suggestion
    'Updates button classes for latest button architecture as of 08/03/2022.\n  - Replaces ds-c-button--primary/success with ds-c-button--solid\n  - Replaces ds-c-button--transparent with ds-c-button--link\n  - Replaces ds-c-button--inverse with ds-c-button--on-dark\n  - Flags ds-c-button and ds-c-button--hover/active/focus as deprecated',
```

Reply:
Strange I thought I commited that fix earlier.